lib/services/groups/groupRegistryMongoDB.js

```diff
@@ -308,6 +308,7 @@ exports.find = alarmsInt(constants.MONGO_ALARM, intoTrans(context, find));
 exports.findType = alarmsInt(constants.MONGO_ALARM, intoTrans(context, findBy(['service', 'subservice', 'type'])));
```

Comment:
CNR entry missing

(I know the PR is WIP, but adding this comment to not miss it...)

Reply:
NTC (already done)